test/blas_test_macros.hpp

```diff
@@ -157,7 +204,8 @@
                                         name_generator);
 #endif  // BLAS_ENABLE_COMPLEX
 
-/** Registers test for all supported data types
+/** Registers test for all supported data
+ * types
```

Comment:
Why this change?

Reply:
Formatting mistake, will revert it!

Reply:
Fixed in [c7aedc9](https://github.com/codeplaysoftware/portBLAS/pull/500/commits/c7aedc965dae663ae2ee9c171f69302c38a4d092).